eng/scripts/InstallVisualStudio.ps1

```diff
@@ -3,14 +3,6 @@
     Installs or updates Visual Studio on a local developer machine.
```

Comment:
This doesn't seem to work:

![image](https://user-images.githubusercontent.com/95136/52224321-91385080-285c-11e9-9c12-ea0dc8fdfef6.png)

Reply:
Yeah those workloads have been removed from VS2019. We need to update korebuild.json to remove them or retarget if they have been renamed.

Reply:
Do you happen to know the new workload names? The docs don't specify the new names https://docs.microsoft.com/en-us/visualstudio/install/workload-and-component-ids?view=vs-2019

Reply:
I tried digging through some metadata. Can't figure out what should replace these workload names. Let me trying sending mail internally.

Reply:
Looks like they're just gone, so no renaming required. I'm also going to update all .vcxproj to use Windows SDK 10.0.17134.0

Reply:
Tested the installvisualstudio.ps1 script locally. Should be working correctly now.